scenarios/microsoft-azure-arcbox-v1/configuration.metadata.json

```diff
@@ -57,6 +57,10 @@
                 "iconType": "CustomIcon",
                 "iconFileName": "AzureMonitor"
             }
+        },
+        {
+            "resourceType": "Microsoft.Network/bastionHosts",
```

Comment:
You override the associated resources in each option. You'll want to mention this in each of those options .. since that's an override. Please add it under the options.